langlib/langlib-test/src/test/resources/test-src/valuelib_test.bal

```diff
@@ -1050,3 +1050,181 @@ function assert(anydata actual, anydata expected) {
         panic e;
     }
 }
+
+///////////////////////// Tests for `requireType()` ///////////////////////////
+
+json  p = {name: "Chiran", age: 24, email: "chirans", height: 178.5, weight: 72.5, property: (), address: [125.0/3, "xyz street",
+{province: "southern", Country: "Sri Lanka"}, 81000], married: false, bloodType: {group: "O", RHD: "+"}};
+
+function testRequireTypeWithInt() returns int|error {
+    int age = check p.age;
+    return age;
+}
+
+function testRequireTypeWithInt2() returns int|error {
+    int height = check p.height;
+    return height;
+}
+
+function testRequireTypeWithInt3() returns int|error {
+    int married = check p.married;
+    return married;
+}
+
+function testRequireTypeWithDecimal() returns decimal|error {
+    decimal height = check p.height;
+    return height;
+}
+
+function testRequireTypeWithDecimal2() returns decimal|error {
+    decimal age = check p.age;
+    return age;
+}
+
+function testRequireTypeWithNil() returns ()|error {
+    () property = check p.property;
+    return property;
+}
+
+function testRequireTypeWithString() returns string|error {
+    string name = check p.name;
+    return name;
+}
+
+function testRequireTypeWithFloat() returns float|error {
+    float weight = check p.weight;
+    return weight;
+}
+
+function testRequireTypeWithUnion1() returns float|int|error {
+    float|int weight = check p.weight;
+    return weight;
+}
+
+function testRequireTypeWithUnion2() returns float|string|error {
+    float|string name = check p.name;
+    return name;
+}
+
+function testRequireTypeWithJson1() returns json|error {
+    json age = check p.age;
+    return age;
+}
+
+function testRequireTypeWithJson2() returns json|error {
+    json height = check p.height;
+    return height;
+}
+
+function testRequireTypeWithJson3() returns json|error {
+    json bloodType = check p.bloodType;
+    return bloodType;
+}
+
+function testRequireTypeWithJson4() returns json|error {
+    json address = check p.address;
+    return address;
+}
+
+function testRequireTypeWithJson5() returns json|error {
+    json weight = check p.weight;
+    return weight;
+}
+
+function testRequireTypeWithJson6() returns json|error {
+    json isMarried = check p.married;
+    return isMarried;
+}
+
+function testRequireTypeWithCast1() returns boolean|error {
+    boolean isMarried = <boolean> check p.married;
+    return isMarried;
+}
+
+function testRequireTypeWithCast2() returns json[]|error {
+    json[] address = <json[]> check p.address;
+    return address;
+}
+
+function testRequireTypeWithCast3() returns map<json>|error {
+    map<json> bloodType = <map<json>> check p.bloodType;
+    return bloodType;
+}
+
+function testRequireType() {
+    decimal h = 178.5;
+    float h1 = 178.5;
+    decimal w = 72.5;
+    json name = "Chiran";
+    json w1 = 72.5;
+    float|int w2 = 72.5;
+    float|string name2 = "Chiran";
+    assert(<int> testRequireTypeWithInt(), 24);
```

Comment:
It seems code is not properly formatted. Remove the  space after `>`

Reply:
Fixed.